packs/october.js

```diff
@@ -0,0 +1,39 @@
+const icon = `data:image/svg+xml,%3Csvg width='310' height='310' xmlns='http://www.w3.org/2000/svg' preserveAspectRatio='xMidYMid meet'%3E%3Cg%3E%3Ctitle%3Eoctober%3C/title%3E%3Crect fill='none' id='canvas_background' height='602' width='802' y='-1' x='-1'/%3E%3C/g%3E%3Cg%3E%3Ctitle%3ELayer 1%3C/title%3E%3Cg id='svg_1' fill='white' transform='translate(0,310) scale(0.10000000149011612,-0.10000000149011612) '%3E%3Cpath fill='%23de6c26' id='svg_2' d='m1350,3031c-142,-44 -349,-218 -499,-420c-293,-396 -455,-869 -437,-1281c8,-202 49,-355 144,-545c136,-274 408,-551 655,-669c141,-67 134,-69 141,28c3,44 10,324 16,621c27,1373 44,1760 82,1913c6,23 14,42 19,42c12,0 36,-91 49,-183c5,-43 10,-177 10,-298c0,-120 3,-219 7,-219c3,0 57,50 119,110c69,68 120,110 133,110c18,0 19,-3 10,-27c-12,-31 -79,-132 -191,-285l-78,-108l1,-312c1,-172 4,-374 8,-449l6,-137l110,126c140,160 351,371 395,396l33,19l-6,-32c-20,-98 -95,-207 -374,-543l-163,-196l0,-311c0,-183 4,-311 9,-311c28,0 193,64 268,105c138,73 238,150 383,295c157,157 244,274 334,452c205,400 210,803 16,1190c-178,356 -546,721 -878,869c-117,53 -246,73 -322,50z'/%3E%3C/g%3E%3C/g%3E%3C/svg%3E`;
```

Comment:
```suggestion
const icon = `data:image/svg+xml,<svg width="310" height="310" xmlns="http://www.w3.org/2000/svg"><path fill="none" d="M-1-1h802v602H-1z"/><g><path fill="%23de6c26" d="M135 6.9c-14.2 4.4-34.9 21.8-49.9 42C55.8 88.5 39.6 135.8 41.4 177c.8 20.2 4.9 35.5 14.4 54.5 13.6 27.4 40.8 55.1 65.5 66.9 14.1 6.7 13.4 6.9 14.1-2.8.3-4.4 1-32.4 1.6-62.1 2.7-137.3 4.4-176 8.2-191.3.6-2.3 1.4-4.2 1.9-4.2 1.2 0 3.6 9.1 4.9 18.3.5 4.3 1 17.7 1 29.8 0 12 .3 21.9.7 21.9.3 0 5.7-5 11.9-11 6.9-6.8 12-11 13.3-11 1.8 0 1.9.3 1 2.7-1.2 3.1-7.9 13.2-19.1 28.5L153 128l.1 31.2c.1 17.2.4 37.4.8 44.9l.6 13.7 11-12.6c14-16 35.1-37.1 39.5-39.6l3.3-1.9-.6 3.2c-2 9.8-9.5 20.7-37.4 54.3L154 240.8v31.1c0 18.3.4 31.1.9 31.1 2.8 0 19.3-6.4 26.8-10.5 13.8-7.3 23.8-15 38.3-29.5 15.7-15.7 24.4-27.4 33.4-45.2 20.5-40 21-80.3 1.6-119-17.8-35.6-54.6-72.1-87.8-86.9-11.7-5.3-24.6-7.3-32.2-5z"/></g></svg>`;
```

Ran through https://jakearchibald.github.io/svgomg/

Reply:
I notice the svg has a fixed size `width="310" height="310"` going to remove that as well.

Reply:
Removing the fixed size `width="310" height="310"` part as well.

Reply:
I tested your svg code and didn't seem to work, plus all the other stack packs seem to have their svg data encoded.

Going to use this:

```
data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='none' d='M-1-1h802v602H-1z'/%3E%3Cg%3E%3Cpath fill='%23de6c26' d='M135 6.9c-14.2 4.4-34.9 21.8-49.9 42C55.8 88.5 39.6 135.8 41.4 177c.8 20.2 4.9 35.5 14.4 54.5 13.6 27.4 40.8 55.1 65.5 66.9 14.1 6.7 13.4 6.9 14.1-2.8.3-4.4 1-32.4 1.6-62.1 2.7-137.3 4.4-176 8.2-191.3.6-2.3 1.4-4.2 1.9-4.2 1.2 0 3.6 9.1 4.9 18.3.5 4.3 1 17.7 1 29.8 0 12 .3 21.9.7 21.9.3 0 5.7-5 11.9-11 6.9-6.8 12-11 13.3-11 1.8 0 1.9.3 1 2.7-1.2 3.1-7.9 13.2-19.1 28.5L153 128l.1 31.2c.1 17.2.4 37.4.8 44.9l.6 13.7 11-12.6c14-16 35.1-37.1 39.5-39.6l3.3-1.9-.6 3.2c-2 9.8-9.5 20.7-37.4 54.3L154 240.8v31.1c0 18.3.4 31.1.9 31.1 2.8 0 19.3-6.4 26.8-10.5 13.8-7.3 23.8-15 38.3-29.5 15.7-15.7 24.4-27.4 33.4-45.2 20.5-40 21-80.3 1.6-119-17.8-35.6-54.6-72.1-87.8-86.9-11.7-5.3-24.6-7.3-32.2-5z'/%3E%3C/g%3E%3C/svg%3E
```

Basically your version that's been encoded and the size attributes removed.

Reply:
@connorjclark 

Looking at all the other stack packs their svg's seem to be `encoded` please confirm if I should use an encoded version or not?

### Non-encoded version

```js
const icon = `data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 310 310"><path fill="none" d="M-1-1h802v602H-1z"/><g><path fill="%23de6c26" d="M135 6.9c-14.2 4.4-34.9 21.8-49.9 42C55.8 88.5 39.6 135.8 41.4 177c.8 20.2 4.9 35.5 14.4 54.5 13.6 27.4 40.8 55.1 65.5 66.9 14.1 6.7 13.4 6.9 14.1-2.8.3-4.4 1-32.4 1.6-62.1 2.7-137.3 4.4-176 8.2-191.3.6-2.3 1.4-4.2 1.9-4.2 1.2 0 3.6 9.1 4.9 18.3.5 4.3 1 17.7 1 29.8 0 12 .3 21.9.7 21.9.3 0 5.7-5 11.9-11 6.9-6.8 12-11 13.3-11 1.8 0 1.9.3 1 2.7-1.2 3.1-7.9 13.2-19.1 28.5L153 128l.1 31.2c.1 17.2.4 37.4.8 44.9l.6 13.7 11-12.6c14-16 35.1-37.1 39.5-39.6l3.3-1.9-.6 3.2c-2 9.8-9.5 20.7-37.4 54.3L154 240.8v31.1c0 18.3.4 31.1.9 31.1 2.8 0 19.3-6.4 26.8-10.5 13.8-7.3 23.8-15 38.3-29.5 15.7-15.7 24.4-27.4 33.4-45.2 20.5-40 21-80.3 1.6-119-17.8-35.6-54.6-72.1-87.8-86.9-11.7-5.3-24.6-7.3-32.2-5z"/></g></svg>`;
```

### Encoded version

```js
data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 310 310'%3E%3Cpath fill='none' d='M-1-1h802v602H-1z'/%3E%3Cg%3E%3Cpath fill='%23de6c26' d='M135 6.9c-14.2 4.4-34.9 21.8-49.9 42C55.8 88.5 39.6 135.8 41.4 177c.8 20.2 4.9 35.5 14.4 54.5 13.6 27.4 40.8 55.1 65.5 66.9 14.1 6.7 13.4 6.9 14.1-2.8.3-4.4 1-32.4 1.6-62.1 2.7-137.3 4.4-176 8.2-191.3.6-2.3 1.4-4.2 1.9-4.2 1.2 0 3.6 9.1 4.9 18.3.5 4.3 1 17.7 1 29.8 0 12 .3 21.9.7 21.9.3 0 5.7-5 11.9-11 6.9-6.8 12-11 13.3-11 1.8 0 1.9.3 1 2.7-1.2 3.1-7.9 13.2-19.1 28.5L153 128l.1 31.2c.1 17.2.4 37.4.8 44.9l.6 13.7 11-12.6c14-16 35.1-37.1 39.5-39.6l3.3-1.9-.6 3.2c-2 9.8-9.5 20.7-37.4 54.3L154 240.8v31.1c0 18.3.4 31.1.9 31.1 2.8 0 19.3-6.4 26.8-10.5 13.8-7.3 23.8-15 38.3-29.5 15.7-15.7 24.4-27.4 33.4-45.2 20.5-40 21-80.3 1.6-119-17.8-35.6-54.6-72.1-87.8-86.9-11.7-5.3-24.6-7.3-32.2-5z'/%3E%3C/g%3E%3C/svg%3E
```

Let me know and then I will add to a commit, thanks.

Reply:
No need to encode any characters here.

> Removing the fixed size width="310" height="310" part as well and converting it to a viewbox to make the svg responsive.

I think this is fine but I asked someone else to weigh in too.

Reply:
I've typically just used `viewBox` and [seems like the we've got expert recommendation here too](https://css-tricks.com/scale-svg/) :)